lib/temporal/testing/local_workflow_context.rb

```diff
@@ -193,6 +194,13 @@ def cancel(target, cancelation_id)
         raise NotImplementedError, 'Cancel is not available when Temporal::Testing.local! is on'
       end
 
+      def upsert_search_attributes(search_attributes)
+        Temporal::Workflow::Context::Private::Validators.validate_search_attributes(search_attributes)
+
+        # We no-op in local testing mode since there is no search functionality.  We don't fail because we 
+        # don't want to block workflows testing other aspects.
```

Comment:
nit: maybe we should still keep a track of the search attributes and allow for mocked `fetch_workflow_execution` calls to return them?

Reply:
Thanks, mind if I punt to the next PR?  I have another fast-follow to do anyway.  Curious if you have any advice on where I should to track the state.  A class variable in `LocalWorkflowContext`, maybe?

Reply:
Sure thing, a follow up PR sounds good.

As to where to place it — the list of executions might be more appropriate than a class variable (which would restrict parallel executions)